source/common/network/connection_impl.cc

```diff
@@ -66,11 +66,8 @@ ConnectionImpl::ConnectionImpl(Event::Dispatcher& dispatcher, ConnectionSocketPt
   }
 
   // Libevent only supports Level trigger on Windows.
```

Comment:
This comment should probably be moved to file_event.h

Reply:
makes sense. I moved it in 1d06ba3e527004c6e03757a3ea37314d9ac8b754